static/styles/settings.scss

```diff
@@ -1271,16 +1291,31 @@ input[type=checkbox] {
             cursor: pointer;
         }
 
+        #realm_icon_delete_button,
         #user_avatar_delete_button {
             opacity: 1;
         }
 
         #user_avatar_upload_button {
             opacity: 1;
         }
+
+        #realm-icon-background {
+            display: block;
+        }
     }
 }
 
+.avatar {
+    width: 200px;
+    height: 200px;
+}
+
+.icon {
+    width: 100px;
+    height: 100px;
+}
```

Comment:
These names are too generic; we display avatars and icons in other places in the app at a range of sizes.  This should have something like `.settings_page_avatar`

Reply:
This seems reasonable:

```
diff --git a/static/styles/settings.scss b/static/styles/settings.scss
index 2eb7f46628..9c45296fff 100644
--- a/static/styles/settings.scss
+++ b/static/styles/settings.scss
@@ -656,6 +656,16 @@ input[type=checkbox] {
     box-shadow: 0px 0px 10px hsla(0, 0%, 0%, 0.1);
 }
 
+#user-settings-avatar {

+    width: 200px;


+    height: 200px;
+}
+
+.realm-icon-section .avatar-icon-settings {
+    width: 100px;
+    height: 100px;
+}
+
 .remove-attachment {
     margin-right: 5px !important;
     font-size: 1.1em !important;
@@ -1347,16 +1357,6 @@ input[type=checkbox] {
     }
 }
 
-.avatar {
-    width: 200px;
-    height: 200px;
-}
-
-.icon {
-    width: 100px;
-    height: 100px;
-}
-
 #user-avatar-source {
     position: absolute;
     font-size: 1em;
diff --git a/static/templates/settings/account_settings.hbs b/static/templates/settings/account_settings.hbs
index 5c2e122651..49b49c5016 100644
--- a/static/templates/settings/account_settings.hbs
+++ b/static/templates/settings/account_settings.hbs
@@ -156,7 +156,7 @@
             <h3>{{t "Profile picture" }}</h3>
 
             <div class="inline-block">
-                <div id="user-settings-avatar" class="avatar-icon-settings avatar">
+                <div id="user-settings-avatar" class="avatar-icon-settings">
                     <div id="user-avatar-background"></div>
                     <div id="user-avatar-block" style="background-image: url({{ page_params.avatar_url_medium }})" {{#if page_params.is_guest}} class="guest-avatar"{{/if}}></div>
                     <span id="user_avatar_delete_button" aria-label="{{t 'Delete profile picture'}}" role="button" tabindex="0"
diff --git a/static/templates/settings/organization_profile_admin.hbs b/static/templates/settings/organization_profile_admin.hbs
index 6768ba97b9..506cda08d1 100644
--- a/static/templates/settings/organization_profile_admin.hbs
+++ b/static/templates/settings/organization_profile_admin.hbs
@@ -28,7 +28,7 @@
         <div>{{t "Organization profile picture" }}</div>
         <div class="realm-icon-section">
             <div class="inline-block">
-                <div class="avatar-icon-settings icon">
+                <div class="avatar-icon-settings">
                     {{#if is_admin}}
                     <div id="realm_icon_upload_button">
                         <div id="realm-icon-background"></div>
```

Reply:
Hereafter I'd consider your feedbacks